### PR TITLE
Added configurable debounce delay and other small stuff

### PR DESCRIPTION
**Pull request by mlohbihler**
- Added configurable debounce delay
- Fixed comment spellings
- Change var name to not conflict with function argument

### CODE REVIEW

File: ThatValidator.js [hunk 1]
```diff
@@ -362,10 +347,10 @@
 
                 for(var i = 0; i < elements.length; i++)
                 {
-                    var field = elements[i];
+                    var field2 = elements[i];
 
                     //if the element has not already been found
-                    if(self.fields.indexOf(field) == -1)
+                    if(self.fields.indexOf(field2) == -1)
```
Comment (sarendipitee) [inline ThatValidator.js]: Not sure about this. Where is the conflict? Where is `field` even defined now to add the handlers?

Comment (mlohbihler) [inline ThatValidator.js]: It's a small thing. "field" is defined in the function arguments, and 
then you define it again with "var field = self.fields[i];". Very 
strictly speaking this isn't allowed.

Comment (sarendipitee) [inline ThatValidator.js]: I think you did it in the wrong place then. I see what you're talking about in the validate() function, but this is in the init() function

Comment (mlohbihler) [inline ThatValidator.js]: Sigh... Lost in translation. I'll fix it.

Sent from Samsung Mobile

<div>-------- Original message --------</div><div>From: Jonathan Dumaine notifications@github.com </div><div>Date:09-14-2015  4:26 PM  (GMT-05:00) </div><div>To: JonDum/ThatValidator ThatValidator@noreply.github.com </div><div>Cc: mlohbihler ml@serotoninsoftware.com </div><div>Subject: Re: [ThatValidator] Added configurable debounce delay and other small
  stuff (#2) </div><div>
</div>In ThatValidator.js:

Comment (mlohbihler) [inline ThatValidator.js]: I fixed the code. Do i need to resubmit the pull request, or will it 
magically update itself?

**Comment by sarendipitee** [comment]
v1.2.0 now on npm

**Comment by mlohbihler** [comment]
Thanks. I've upgraded the code.

**Pull request merged**